check-before-deployment.md

```diff
@@ -112,13 +112,7 @@ Take the `/dev/nvme0n1` data disk as an example:
 
 ## Check and disable system swap
 
-This section describes how to disable swap.
-
-TiDB requires sufficient memory space for operation. It is not recommended to use swap as a buffer for insufficient memory, which might reduce performance. Therefore, it is recommended to disable the system swap permanently.
-
-Do not disable the system swap by executing `swapoff -a`, or this setting will be invalid after the machine is restarted.
-
-To disable the system swap, execute the following command:
+TiDB requires sufficient memory space for operation. Using swap as a buffer when memory is insufficient might deteriorate performance. Therefore, it is recommended to disable the system swap permanently by executing the following command:
```

Comment:
Please translate literally and try to use simple words. "deteriorate" seems a misuse here.